lib/api-group.js

```diff
@@ -23,6 +28,10 @@ class ApiGroup {
     this.requestOptions.cert = options.cert;
     this.requestOptions.key = options.key;
 
+    if (options.hasOwnProperty('insecureSkipTlsVerify')) {
+      this.requestOptions.strictSSL = !options.insecureSkipTlsVerify;
```

Comment:
Can we utilize `options.request` for this? Then when constructing the object we use:
```
const k8 = new K8Api({
   url: 'https://my-k8-api-server.com',
   request: {
    strictSSL: false
   },
   auth: {
     user: 'user',
     pass: 'pass'
   }
 });
```